local block adaptor import support (#5277)
treeverse/lakeFS: docs/reference/configuration.md

   If specified, the storage namespace will be filled with this default value as a prefix when creating a repository from the UI.
   The user may still change it to something else.
* `blockstore.local.path` `(string: "~/lakefs/data")` - When using the local Block Adapter, which directory to store files in
* `blockstore.local.import_enabled` `(bool: false)` - Enable import for local Block Adapter, relevant if you are using shared location

> * `blockstore.local.allowed_external_prefixes` `([]string: [])` - List of prefixes used to match any access for external location used by local block adapter

Author: Updated the description - let me know if it is clear now.

> * `blockstore.local.allowed_external_prefixes` `([]string: [])` - List of prefixes used to match any access for external location used by local block adapter

Reviewer: Did you push?

   If specified, the storage namespace will be filled with this default value as a prefix when creating a repository from the UI.
   The user may still change it to something else.
* `blockstore.local.path` `(string: "~/lakefs/data")` - When using the local Block Adapter, which directory to store files in
* `blockstore.local.import_enabled` `(bool: false)` - Enable import for local Block Adapter, relevant only if you are using shared location
* `blockstore.local.import_hidden` `(bool: false)` - When enabled import will scan and import any file or folder that starts with a dot character.

> * `blockstore.local.allowed_external_prefixes` `([]string: [])` - List of absolute path prefixes used to match any access for external location by the local block adapter (ex: /var/data/).

Reviewer: ```suggestion
* `blockstore.local.allowed_external_prefixes` `([]string: [])` - List of absolute path prefixes that are allowed to be imported. (ex: /var/data/).
```
